index.js

```diff
@@ -66,16 +65,16 @@ queue.on('job failed', function(){
 queue.watchStuckJobs();
 
 process.once('uncaughtException', function(err){
-  queue.shutdown(function(err2){
+  queue.shutdown(9500, function(err2){
```

Comment:
Soo, at the end @TheHollidayInn code got merged in so we're using the last version of Kue and I think this queue.shutdown syntax is the old one. I'm confused though, so tell me if I'm wrong. This is the syntax used now in master https://github.com/HabitRPG/habitrpg-email-server/blob/master/index.js#L69